urbanopt_des/modelica_results.py

```diff
@@ -271,12 +292,16 @@ def resample_and_convert_to_df(
         df_energy.index.name = "datetime"
 
         # all data combined
-        data = {
-            "datetime": time,
-            "Sewer Pump Electricity": sewer_pump,
-            "GHX Pump Electricity": ghx_pump,
-            "Distribution Pump Electricity": distribution_pump,
-        } | building_data
+        data = (
+            {
```

Comment:
Why the new parentheses? What is that doing?

Reply:
because of the pipes combining the data. But maybe they aren't needed?

Reply:
Those are `or` pipes? Humm. Not sure if they're necessary or not. We should test this! 😄 Then I can play with removing the parens.

Reply:
well, funny. I remove the () and then moved up the pipes and it passed. Then ran ruff and it added back in the (). So they are just for formatting to make it look better!